packages/clients/src/scw/fetch/types.ts

```diff
@@ -5,6 +5,7 @@ export type ScwRequest = {
   headers?: Record<string, string>
   body?: string
   urlParams?: URLSearchParams
+  responseType?: 'JSON' | 'TEXT' | 'BLOB'
```

Comment:
Uppercase is usually only used for constants, I think it would make more sense to keep same in lowercase.